cmd/kube-apiserver/app/apiextensions.go

```diff
@@ -39,6 +39,13 @@ func createAPIExtensionsConfig(kubeAPIServerConfig genericapiserver.Config, exte
 	etcdOptions.StorageConfig.Codec = apiextensionsapiserver.Codecs.LegacyCodec(v1beta1.SchemeGroupVersion)
 	genericConfig.RESTOptionsGetter = &genericoptions.SimpleRestOptionsFactory{Options: etcdOptions}
 
+	if err := commandOptions.APIEnablement.ApplyTo(
```

Comment:
add a comment: // override MergedResourceConfig with apiextensions defaults and registry